actionlist/app/actions/action_items.js

```diff
@@ -0,0 +1,6 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
```

Comment:
This directive shouldn't be needed with the recent changes to `.eslintrc`. Does the build fail without it?